src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Keyword-Strings.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "NAMES": ["burning"],
-    "DESCRIPTION": "Burning creatures take damage at the start of their turn and have healing reduced by 50%. Each turn, Burning is reduced by half."
+    "DESCRIPTION": "Burning creatures take damage at the start of their turn and have healing prevented. Each turn, Burning is reduced by a third."
```

Comment:
I feel like "have healing prevented" sounds a little ambiguous about whether it prevents them from healing others; maybe "cannot be healed" would be more clear, more like how you've update the power strings?

Reply:
Sure, that makes sense.